lib/ddtrace/contrib/active_support/cache/instrumentation.rb

```diff
@@ -105,6 +158,31 @@ def fetch(*args, &block)
             end
           end
 
+          # Defines instrumentation for ActiveSupport cache fetching of multiple keys
+          module FetchMulti
+            def fetch_multi(*keys, &block)
+              # extract options hash
+              keys.pop if keys[-1].instance_of?(Hash)
```

Comment:
`#pop` is mutating the argument array, thus affecting the arguments passed to the `super` call, is this expected?

I believe this would change the semantics of `cache_store_class(:fetch_multi)#fetch_multi`.

Reply:
Good catch! 👍 I didn't pay enough attention to it indeed. Fixed.